src/index.html

```diff
@@ -11,7 +11,136 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="label">
+          How old are You?:
+          <input type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input type="data"
+            name="birth"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="label">
+          I accept the terms of the agreement
+          <input type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input type="email"
+            name="e-mail"
+            placeholder="email@example.com"
+            required
+           >
+        </label>
+        <label class="label">
+          Password:
+          <input type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          Do you love cats?
+          <input type="radio"
+            name="cats"
+            value ="yes"
+          >Yes
+          <input type="radio"
+            name="cats"
+            value ="no"
+          >No
+        </label>
```

Comment:
https://mateacademy.slack.com/archives/C03REBVA3AN/p1660653742562429

```suggestion
        <label class="label">
          Do you love cats?
          <input type="radio"
            name="cats"
            value ="yes"
          >
        	Yes
          <input type="radio"
            name="cats"
            value ="no"
          >
          No
        </label>
```

Reply:
ok